.goreleaser.yml

```diff
@@ -178,6 +225,23 @@ docker_manifests:
     image_templates:
       - diamon/helmwave:latest-amd64
       - diamon/helmwave:latest-arm64v8
+  - name_template: ghcr.io/helmwave/helmwave:{{ .Version }}-debug
+    image_templates:
+      - ghcr.io/helmwave/helmwave:{{ .Version }}-debug-amd64
+      - ghcr.io/helmwave/helmwave:{{ .Version }}-debug-arm64v8
+  - name_template: ghcr.io/helmwave/helmwave:latest
```

Comment:
it should be `latest-debug`

Reply:
My first idea was to use `latest-debug`

But I saw other projects using a simple `debug` tag

Reply:
Okay, so lets drop this one then because you added `debug` tag a bit lower in the code

Reply:
What about fixed debug tags?

Reply:
I think it's okay to leave other as is